web/v2/backend/src/main/java/com/tourneygen/web/Models/DTOs/TeamUpdateDTO.java

```diff
@@ -0,0 +1,116 @@
+package com.tourneygen.web.Models.DTOs;
+
+import com.google.gson.Gson;
+import com.tourneygen.web.Models.Team;
+import java.util.Set;
+import javax.validation.constraints.Min;
+import javax.validation.constraints.NotNull;
+import javax.validation.constraints.Size;
+
+public class TeamUpdateDTO {
+
+  @NotNull private Long id;
+
+  private Set<String> roster;
+
+  @Min(value = 0)
+  private int wins = 0;
+
+  @Min(value = 0)
+  private int losses = 0;
+
+  private Long owner;
+
+  @Size private String name;
+
+  @Size private String description;
+
+  private Long league;
+
+  private float rating;
+
+  public Long getId() {
+    return id;
+  }
+
+  public void setId(Long id) {
+    this.id = id;
+  }
+
+  public Set<String> getRoster() {
+    return roster;
+  }
+
+  public void setRoster(Set<String> roster) {
+    this.roster = roster;
+  }
+
+  public int getWins() {
+    return wins;
+  }
+
+  public void setWins(int wins) {
+    this.wins = wins;
+  }
+
+  public int getLosses() {
+    return losses;
+  }
+
+  public void setLosses(int losses) {
+    this.losses = losses;
+  }
+
+  public Long getOwner() {
+    return owner;
+  }
+
+  public void setOwner(Long owner) {
+    this.owner = owner;
+  }
+
+  public String getName() {
+    return name;
+  }
+
+  public void setName(String name) {
+    this.name = name;
+  }
+
+  public String getDescription() {
+    return description;
+  }
+
+  public void setDescription(String description) {
+    this.description = description;
+  }
+
+  public Long getLeague() {
+    return league;
+  }
+
+  public void setLeague(Long league) {
+    this.league = league;
+  }
+
+  public String toJson() {
+    return new Gson().toJson(this, TeamUpdateDTO.class);
+  }
+
+  public TeamUpdateDTO fromJson(String src) {
+    return new Gson().fromJson(src, TeamUpdateDTO.class);
+  }
+
+  public TeamUpdateDTO() {}
```

Comment:
Is this a pattern I'm not familiar with? or a typo?

Reply:
pattern DTO stand fro data transfer object

Reply:
Awesome, just making sure!